forte/ontology_specs/payload_ontology.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
If `JpegPayload` and `SoundFilePayload` are only for test cases, then maybe it's better to put them under [test_specs](https://github.com/asyml/forte/tree/master/tests/forte/data/ontology/test_specs) and then generate [payload_ontology.py](https://github.com/asyml/forte/pull/859/files#diff-bb87b290212795bb61b27c174b0f921e341811878d71940897c60202c47b1dec) under [test_outputs](https://github.com/asyml/forte/tree/master/tests/forte/data/ontology/test_outputs).

Reply:
I think it could be use for other tasks later as I will need to write a training pipeline.